src/test/scala/outwatch/DomEventSpec.scala

```diff
@@ -418,8 +419,11 @@ class DomEventSpec extends JSDomSpec {
 
     document.getElementById("input").dispatchEvent(inputEvt)
 
-    winClicked shouldBe true
-    docClicked shouldBe true
+    Task {
```

Comment:
Could you please explain, why this is neccessary here?

Reply:
This is because Monix uses by default a [batched execution model](https://monix.io/docs/2x/execution/scheduler.html#execution-model), so we need to "fork" the assertion to the end of the javascript execution queue, otherwise it might execute before all the effects of the event dispatch execute.